### PR TITLE
Update vulnerability-to-poodle-attack.md

### PR DESCRIPTION
**Pull request by skibz**

### CODE REVIEW

File: security/ssl/vulnerability-to-poodle-attack.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The POODLE attack is an exploit that relies on the presence of legacy security schemes for interoperability and backwards-compatibility.
```
Comment (Johanndutoit) [inline security/ssl/vulnerability-to-poodle-attack.md]: I had to focus when reading  that, think you'll blow someone like Jacques mind when you say that. Thinking we need to remove the big words and also give a example so anyone can put this in context to show how bad this is.

**Comment by skibz** [comment]
i've amended the opening paragraph, but this document still needs a lot of work. there's about three variants of poodle. initially, it was an exploit that downgraded the connection to sslv3, then a new one was reported that could exploit tls 1.0 - 1.2 connections, too. i've had difficulty researching the latter.

**Comment by Johanndutoit** [comment]
Yeah think this could go on forever. Think to just get a initial copy out just add what it allows a attacker to do in a second paragraph. Should be enough for now, I'd rather link off to detailed documents than make this readme too big. Thoughts ?

**Comment by Johanndutoit** [comment]
Nice, thanks for the update. LGTM

**Pull request merged**